src/Reflection/ReflectionType.php

```diff
@@ -22,30 +33,15 @@ private function __construct()
     {
     }
 
-    /**
-     * @param Type $type
-     * @param bool $allowsNull
-     * @return ReflectionType
-     */
-    public static function createFromType(Type $type, bool $allowsNull) : self
+    public static function createFromType(string $type, bool $allowsNull) : self
```

Comment:
This is a BC break, as this thing was [already in `1.2.0`](https://github.com/Roave/BetterReflection/blob/5d656518b027d3303dee197cec10a7fcb5b712ab/src/Reflection/ReflectionType.php#L29)